.bin/create-rust-webpack.js

```diff
@@ -32,10 +32,17 @@ if (!isPresent("git --version")) {
   process.exit(1);
 }
 
+const args = process.argv.slice(2);
+
+if (args.some(arg => arg.includes('-'))) {
+  console.log(`Error: unknown option ${args[0]}`);
+  process.exit(1);
+}
+
 let folderName = '.';
 
-if (process.argv.length >= 3) {
-  folderName = process.argv[2];
+if (args.length >= 3) {
```

Comment:
Are you sure about this? At a first glance `args.length >= 3` seems wrong. Can you double check?

Reply:
oops!